tools/chat_helpers/channel_msg_forces.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+require_once(TOOLS . 'chat_helpers/channel_msg_seedlist.php');
+
+function shared_channel_msg_forces($player, $option) {
+	$db = new SmrMySqlDatabase();
+	if (!empty($option) && $option == "seedlist") {
```

Comment:
If you did
```
if (empty($option)) {
	...
}
else if ($option == 'seedlist') {
	...
}
else {
	...
}
```
then you could avoid having to check the !empty case twice

Reply:
Thanks for the suggestion. I've made this change.